package.json

```diff
@@ -44,6 +44,9 @@
   "config": {
     "commitizen": {
       "path": "./node_modules/cz-conventional-changelog"
+    },
+    "publish": {
+      "message": "chore(release): publish %s"
```

Comment:
Quoting from https://github.com/lerna/lerna#--message--m-msg, the emphasis is mine:
> If the message contains %s, it will be replaced with the new global version version number prefixed with a "v". If the message contains %v, it will be replaced with the new global version version number without the leading "v". **Note that this only applies when using the default "fixed" versioning mode, as there is no "global" version when using --independent.**

AFAIK, we are using the independent mode, therefore there is no single version to substitute for `%s`.

I am proposing to change the message to `chore(release): publish` as is shown in Lerna docs.

```bash
$ lerna publish -m "chore(release): publish" --independent
# commit message = "chore(release): publish
#
# - package-1@3.0.1
# - package-2@1.5.4"
```